books-and-breakfast/app/SchoolTransportDetails.tsx

```diff
@@ -88,4 +156,23 @@ const styles = StyleSheet.create({
     textAlign: 'left',
     lineHeight: 23,
   },
+  editText: {
```

Comment:
I think setting the heights for editText and text to be the same helps avoid the glitch issue. Maybe add padding or margin so that the pre-edit and post edit height are the same?